### PR TITLE
Make the planner path a line instead of points

### PR DESCRIPTION
**Pull request by adamantivm**

### CODE REVIEW

**Comment by adamantivm** [comment]
This makes it possible to see at different zoom levels.
@jubeira ptal

File: android_15/src/org/ros/android/view/visualization/layer/PathLayer.java [hunk 1]
```diff
@@ -79,27 +81,21 @@ public void onNewMessage(nav_msgs.Path path) {
 
   private void updateVertexBuffer(nav_msgs.Path path) {
     ByteBuffer goalVertexByteBuffer =
-        ByteBuffer.allocateDirect(path.getPoses().size() * 3 * Float.SIZE / 8);
```
Comment (jubeira) [inline android_15/src/org/ros/android/view/visualization/layer/PathLayer.java:82]: Was this allocating twice as much as required before, or am I getting it wrong?

**Review by jubeira — APPROVED** [review]
LGTM

File: android_15/src/org/ros/android/view/visualization/layer/PathLayer.java (hunk 1 above)
Comment (adamantivm) [reply android_15/src/org/ros/android/view/visualization/layer/PathLayer.java:82]: Right. I think that logic was wrong, probably due to the lack of using a constant for the separation between points.

**Pull request merged**